### ISSUE TITLE
chunk data when running in parallel

### ISSUE THREAD

**Issue by andportnoy** [kutaslab/fitgrid#67]
`multiprocessing` doesn't allow for passing objects more than 2Gb in size:
https://bugs.python.org/issue17560

This was fixed and merged into master:
https://github.com/python/cpython/commit/bccacd19fa7b56dcf2fbfab15992b6b94ab6666b

But Python 3.8 is only scheduled to be released in the end of 2019:
https://www.python.org/dev/peps/pep-0569/#schedule

So for now we need to chunk the data into < 2Gb pieces.

**Comment by andportnoy** [comment]
See #98.
